src/app/main/component/events/components/event-editor/event-editor.component.spec.ts

```
@@ -1,0 +1,96 @@
+import { HarnessLoader } from "@angular/cdk/testing";
```

ESLint

[error] Strings must use singlequote.

(quotes)